scripts/render_platform_manifests.py

```
@@ -0,0 +1,365 @@
+    command_name : str
+        Human-readable name for error messages (e.g., "init", "apply").
+    args : list[str]
+        Command arguments to pass to run_tofu.
+    module_path : Path
+        Working directory for the command.
+    """
+    print(f"\n--- Running tofu {command_name} ---")
+    result = run_tofu(args, module_path)
+    if not result.success:
+        print(f"error: tofu {command_name} failed: {result.stderr}", file=sys.stderr)
+        raise RuntimeError(f"tofu {command_name} failed")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)